### ISSUE TITLE
Not compatible with Symfony 4.1

### ISSUE THREAD

**Issue by tristanbes** [tehplague/swiftmailer-mailgun-bundle#63]
I'm migrating a big app from 3.4 to 4.1 and this packages gives me:

```
cspoo/swiftmailer-mailgun-bundle dev-master requires symfony/force-lowest ^3 -> satisfiable by symfony/force-lowest[v3.4.0].
    - Conclusion: don't install symfony/symfony v4.1.1
```

Tried `composer update` and `composer update --with-dependencies`

**Comment by Nyholm** [comment]
That dependency must be a mistake. It should only be adde on Travis. Could you remove it?

**Comment by tristanbes** [comment]
Sure, it's done : 

#64 (with bonus #65 and #66)

**Comment by Nyholm** [comment]
Closed by #67
